R/JaBbA.R

```diff
@@ -55,13 +55,28 @@ low.count=high.count=seg=chromosome=alpha_high=alpha_low=beta_high=beta_low=pred
         jmessage("${CPLEX_DIR}/cplex/[(include)|(lib)] do not both exist")
     }
 
-    if (!requireNamespace("gurobi", quietly = TRUE)) {
-        jmessage("Gurobi is not installed! REMEMBER: You need to have either CPLEX or Gurobi!")
+    if (((!file.exists(paste0(cplex.dir, "/cplex"))) & (!file.exists(paste0(cplex.dir, "/cplex/include")) ||
```

Comment:
Might be good to store these booleans into variables with informative names (e.g `cplex_exists`, `gurobi_exists` etc.). This is will make these conditionals much easier to read and reduce clutter (since you're repeating this code several times for subsequent conditionals).